fission-web-server/library/Fission/Web/Server/Internal/Production.hs

```diff
@@ -118,11 +119,32 @@ runInProd overrideVerbose action = do
 
   putStrLnIO "   🕷️  Setting up HTTP manager"
   putStrLnIO "      🎛️  Configuring..."
-  httpManager <- HTTP.newManager HTTP.defaultManagerSettings
-                   { HTTP.managerResponseTimeout = HTTP.responseTimeoutMicro clientTimeout }
 
-  putStrLnIO "      ✨ Creating manager"
-  tlsManager  <- HTTP.newManager HTTP.tlsManagerSettings
+  let
+    httpSettings = HTTP.defaultManagerSettings
+      { HTTP.managerResponseTimeout = HTTP.responseTimeoutMicro clientTimeout
+      , HTTP.managerConnCount       = 200
```

Comment:
Each client HTTP manager can handle a certain number of open connections. The default is 10, which obviously won't work for us when we're making parallel requests to 5+ nodes per request. Bumped to 200.

The docs say that we should prefer one manager when possible (I think for efficiency and setting an upper limit on resource use). We can always create these in the cluster handler, but for now 200 (~40 concurrent requests) feels pretty reasonable.

Reply:
Broke out one for IPFS to handle the IPFS timeout for us. This may be unnecessary, but we probably want that as a separate dial from, say, making email requests.

Reply:
ah okay :ok_hand: you answered my question from another comment